api/app/controllers/spree/api/v2/base_controller.rb

```diff
@@ -4,6 +4,8 @@ module V2
       class BaseController < ActionController::API
         include CanCan::ControllerAdditions
 
+        rescue_from ActiveRecord::RecordNotFound, with: :render_not_found_response
```

Comment:
I think `resource_not_found` better describes this concept.

Reply:
Yeah, I agree 👍